Introspect/ViewExtensions.swift

```diff
@@ -198,5 +198,10 @@ extension View {
     public func introspectTabView(customize: @escaping (NSTabView) -> ()) -> some View {
         return introspect(selector: TargetViewSelector.siblingContaining, customize: customize)
     }
+    
+    /// Finds a `NSButton` from a `SwiftUI.Button`
+    public func introspectButton(customize: @escaping (NSButton) -> ()) -> some View {
+        return introspect(selector: TargetViewSelector.siblingContaining, customize: customize)
```

Comment:
I think this should be constrained to `macOS` since it doesn't work on `iOS`.

Reply:
There is an #if canImport(AppKit) check around the entire extension and no other method has the change you propose.